Reworked README.md

DoULikeCookies opened the pull request:
requirements.txt, CONTRIBUTING.md and ACKNOWLEDGMENTS.md are now unnecessary. Put it all in the Readme file and made that look more beautiful.

PolarBear4u commented:
readme.md is lookin good! 

requirements.txt, CONTRIBUTING.md and ACKNOWLEDGMENTS.md as separated files is still better. For example when you clone the repository with git and you want to install all the dependencies, you would type something like pip install -r requirements.txt
Thus, those files are still needed.

PS: you wrote my name wrong (in the readme file)

DoULikeCookies commented:
Added requirements.txt, CONTRIBUTING.md and ACKNOWLEDGMENTS.md as separated files. Updated the Readme a bit again and wrote your name right this time :D

PolarBear4u commented:
great. merged.
thank you very much!
 
im going to fix some minor problems
